src/beamlime/dashboard/__init__.py

```diff
@@ -1,2 +1,5 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) 2025 Scipp contributors (https://github.com/scipp)
+import holoviews as hv
+
+hv.Dimension.unit_format = ' [{unit}]'
```

Comment:
Since this is a global setting, I think it should be set in the startup routine, not on import. This is to ensure that it runs in a predictable manner and only when desired.